doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.html

```diff
@@ -2,8 +2,6 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 	<head>
 		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
-		<title>19.1. Core Mechanism and Applications</title>
```

Comment:
Oups, you delete those line.
Can you generate with build.xml as described here
https://github.com/eclipse/capella/wiki/Development-How-To#generate-html-documentation

Reply:
ohh sorry thank you